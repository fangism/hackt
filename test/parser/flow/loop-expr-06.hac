// "loop-expr-06.hac"
// copy-modified from "loop-expr-03.hac"
// testing loop expressions -- expect fail

pint W[4];
W = {0xf000, 0xd, 0xee0, 0x0};

pint Z = (||:i:4: W[i]);	// reject: || is boolean-OR

