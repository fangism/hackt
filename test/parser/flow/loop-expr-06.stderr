Error: Use '||' for logical-OR on pbools.  [8:11]
Error in source expression.
HALT: at least one error in the assignment list.  [8:8..23]
